python/lsst/pipe/base/pipelineIR.py

```diff
@@ -191,18 +193,20 @@ class ParametersIR:
     task configuration blocks to specify configuration values. They may not be
     used in the special ``file`` or ``python`` blocks.
 
-    Example:
-    paramters:
-      shared_value: 14
-    tasks:
-      taskA:
-        class: modA
-        config:
-          field1: parameters.shared_value
-      taskB:
-        class: modB
-        config:
-          field2: parameters.shared_value
+    Examples
+    --------
+    .. code-block:: yaml
+        parameters:
+        shared_value: 14
```

Comment:
This line needs more indentation? And tasks in `tasks:` below too. And contents of `config:`.